AdGoBye/PluginLoader.cs

```diff
@@ -76,16 +73,51 @@ private static void LoadPlugin(string pluginPath)
             }
 
             if (!string.IsNullOrEmpty(pluginName) && !string.IsNullOrEmpty(pluginVersion) &&
-                pluginClass != null) break;
+                pluginClass is not null) break;
         }
 
         if (string.IsNullOrEmpty(pluginName) || string.IsNullOrEmpty(pluginVersion) ||
-            pluginClass == null)
+            pluginClass is null)
         {
             Logger.Error("Plugin {path} failed to load as it was missing a name, version or class.", pluginPath);
             return;
         }
 
+#if !DEBUG
+        if (!Settings.Options.DisablePluginInstallWarning)
+        {
+            var allowlist = LoadPluginAllowlist();
+            if (!allowlist.Contains($"{pluginName} ({pluginMaintainer}, {pluginVersion})"))
+            {
+                Logger.Information("""
```

Comment:
Maybe add a separator instead of spaces?
Maybe like `----...` or `===...`

Reply:
Are you suggesting literally something like this for the output?
```suggestion
                Logger.Information("""==================
```

Reply:
Yup, at the end as well